src/components/Footer.jsx

```diff
@@ -58,6 +58,7 @@ export default class Footer extends Component {
                 <p className="footerCopyright">
                     <a href="https://github.com/bitcoinz-sites/paper.btcz.rocks" target="_blank" rel="noopener noreferrer">JavaScript Client-Side BitcoinZ Wallet Generator</a>
                 </p>
+                <div className="text-center"><h3><span className="label label-danger text-center">Beta</span></h3></div>
```

Comment:
Can you add a "please test and become familiar with your paper wallet before transferring large sums. Also, please ensure you have adequate backups so as to not lose your only source to your offline coins."

Something along those lines..?

Reply:
updated in new commit